src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_img2img.py

```diff
@@ -284,7 +284,10 @@ def __call__(
 
         # Some schedulers like PNDM have timesteps as arrays
         # It's more optimzed to move all timesteps to correct device beforehand
-        timesteps_tensor = torch.tensor(self.scheduler.timesteps[t_start:], device=self.device)
+        if torch.is_tensor(self.scheduler.timesteps):
```

Comment:
`self.scheduler.timesteps` should always be a tensor - I don't think we should have these if statements here (netiher in `pipeline_stable_diffusion.py` BTW)

@kashif @anton-l could we make sure that `self.scheduler.timesteps` is always of type `torch`

Reply:
ok... since its always used for the tqdm counter I never converted it to torch in the torch-only PR... should i make it always a torch tensor?

Reply:
Ah yes I think we should always make it a PyTorch tensor now that schedulers are PyTorch-only . Think `tqdm` should be a 2nd citizen here :-)

Reply:
I see... the PR now removes the need for the move to pytorch tensor and all seems to work too... but yes all torch tensors would also work similarly then...